### DISCUSSION TITLE
Trying to use JeelizResizer for Canvas and video resolutions

### DISCUSSION THREAD

**Discussion by elifares** [jeeliz/jeelizFaceFilter#220] [Q&A] [answered]
I'm following the instructions for [Canvas and video resolutions](https://github.com/jeeliz/jeelizFaceFilter#canvas-and-video-resolutions) but when I add the code, I get an error saying `SORRY BRO :( . ERR = NO_CANVASID` it seems like `JEEFACEFILTERAPI.init` is not getting the `canvasId`.

Should I put `canvasId: 'jeeFaceFilterCanvas',` inside both `JeelizResizer.size_canvas` and `JEEFACEFILTERAPI.init`?

**Comment by xavierjs** [comment, marked as answer]
You should have a canvas whose `id='jeeFaceFilterCanvas' ` in the DOM.
